swift/StableDiffusion/pipeline/StableDiffusionXLPipeline.swift

```diff
@@ -49,16 +52,18 @@ public struct StableDiffusionXLPipeline: StableDiffusionPipelineProtocol {
     ///   - reduceMemory: Option to enable reduced memory mode
     /// - Returns: Pipeline ready for image generation
     public init(
-        textEncoder: TextEncoderXLModel,
+        textEncoder: TextEncoderXLModel?,
         textEncoder2: TextEncoderXLModel,
         unet: Unet,
+        unetRefiner: Unet?,
```

Comment:
We will need to support chunked `unetRefinerChunk1` and `unetRefinerChunk2` as well

Reply:
Let me drop a final task list from my side and resolve this for now